.gitlab-ci.yml

```diff
@@ -325,389 +344,21 @@ before_script:
 # stage: deploy-part2
   MOVED_TO_GROUP_VARS: "true"
 
-
-# Builds for PRs only (premoderated by unit-tests step) and triggers (auto)
-### PR JOBS PART1
-
-gce_ubuntu18-flannel-aio:
+## Packet-CI
+packet_ubuntu18-flannel-aio:
   stage: deploy-part1
   <<: *job
-  <<: *gce
```

Comment:
it doesn't have to be one or the other, I would keep all gce and add the packet one